modules/health_quest/app/controllers/health_quest/v0/pgd_questionnaires_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  module V0
+    class PgdQuestionnairesController < HealthQuest::V0::BaseController
+      def show
+        ques = questionnaire_by_id
```

Comment:
I think there could be a similar refactoring here https://github.com/department-of-veterans-affairs/vets-api/pull/5001/files#r497520063

Reply:
Refactored as requested.